source/macos/AdaptiveCards/AdaptiveCards/Utils/HostConfigUtils.swift

```diff
@@ -123,3 +123,25 @@ class ColorUtils {
         }
     }
 }
+
+class MarkdownUtils {
```

Comment:
Suggestion: We can call it `TextUtils` instead of Markdown Utils as its more generic.

Reply:
Yeah makes sense
Done!